src/auth.rs

```diff
@@ -91,6 +103,10 @@ impl Auth {
         &self,
         external_id: String,
     ) -> Result<String, Error> {
+        if external_id.is_empty() {
+            return Err(Error::Other("external_id is required".to_string()));
```

Comment:
Looks like this should be
```suggestion
            return Err(Error::InvalidArgument("external_id is required".to_string()));
```

Reply:
😞 [5616f12](https://github.com/passageidentity/passage-flex-rust/pull/34/commits/5616f12042a14d52825a254cda2e7f6dadf31d0c)